### DISCUSSION TITLE
Release of New R Package: OSPSuite.Addins

### DISCUSSION THREAD

**Discussion by pchelle** [Open-Systems-Pharmacology/Forum#1855] [Announcement]
We are happy to announce the first release of RStudio Addins for the OSPSuite R ecosystem ! &#x1f389;

Please visit the __OSPSuite.Addins__ website [&#x1f517;](https://www.open-systems-pharmacology.org/OSPSuite.Addins) to check out its features, including a [__unit converter__](https://www.open-systems-pharmacology.org/OSPSuite.Addins/articles/unit-converter.html) and a [__simulation tree browser__](https://www.open-systems-pharmacology.org/OSPSuite.Addins/articles/simulation-tree.html) !

You can install the package using the code below: 

```r
# install.packages("remotes")
remotes::install_github("Open-Systems-Pharmacology/OSPSuite.Addins")
```

**Comment by tobiasK2001** [comment]
WOW! that really seems to be super useful! Thanks for that!
